### PR TITLE
Feature/create badge notification

### PR DESCRIPTION
**Pull request by ruben-martin-lozano**
Hi @SUI-Components/developers we've created the badge-counter component.

The documentation for this component is still WIP, but as accorded with UX team (Nilo & Cris), we've started to develop it with Zeplin definition as we need it for FC asap.

image

https://app.zeplin.io/project/57175ac73411f50f35b16bd6/screen/5a0d783aae5fa715ab539a6f

### CODE REVIEW

**Review by davidbarna — APPROVED** [review]
Cant' access the zepplin but surprive to see a counter without a number.
Is it for a fake AB test ??

**Comment by ruben-martin-lozano** [comment]
Hi @davidbarna! This is not for an AB test. From the UX point of view, this componente (badge-counter) can contain nothing, a number (from 1-9), and a "dot". In FC we implement it as a "dot" (as Zeplin definition), and because there's no complete definition for this component, that's the reason why there's only the code for the "dot" version :)

File: components/atom/badgeCounter/src/index.scss [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      border-width: $border-size-badge-counter-medium;
+    }
+
+    &-bullet-large {
+      border-width: $border-size-badge-counter-large;
+    }
+  }
+
+  &-size {
+    &-small {
```
Comment (JoanClaret) [inline components/atom/badgeCounter/src/index.scss]: in order to be more consistent with other components, i would define classnames following SUIT naming convention: https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md
for instance: 

```SCSS
.sui-BadgeCounter--small {} // size
.sui-BadgeCounter--thin {} // border
```

how does it looks to you?

Comment (ruben-martin-lozano) [reply components/atom/badgeCounter/src/index.scss]: 👍

**Comment by ruben-martin-lozano** [comment]
Hi @JoanClaret! Feedback applied :)

**Review by JoanClaret — APPROVED** [review]

**Pull request merged**